charts/shoot-core/components/charts/node-problem-detector/templates/daemonset.yaml

```diff
@@ -98,6 +101,9 @@ spec:
           hostPath:
             path: /etc/localtime
             type: "FileOrCreate"
+        - name: kmsg
```

Comment:
I think, the PSP has to be adapted to this change, right?
https://github.com/gardener/gardener/blob/78ef5841ad5a0d9ba405a5ccb66b0e9869a9f390/charts/shoot-core/components/charts/node-problem-detector/templates/psp.yaml#L25-L27

Reply:
huh, thanks for pointing out